geode-core/src/main/java/org/apache/geode/internal/lang/SystemPropertyHelper.java

```diff
@@ -32,33 +30,30 @@
 
   /**
    * Setting this to "true" enables a new asynchronous eviction algorithm. For more details see
```

Comment:
This comment is a bit misleading now since it defaults to true.

Reply:
Updated text to indicate this now defaults to true